docs/scripting.md

```diff
@@ -380,7 +380,7 @@ The most common use of this is for providing HTTP end points for services with w
 
 ```coffeescript
 module.exports = (robot) ->
-  robot.router.post '/hubot/chatsecrets/:room', (req, res) ->
+  robot.router.post '/hubot/chatsecrets/:room_name_or_token', (req, res) ->
     room   = req.params.room
```

Comment:
What @michaelansel is pointing out is the route uses `room_name_or_token`, but then accesses `req.params.room`. It should be updated `req.params.room_name_or_token`.